docs/source/conf.py

```diff
@@ -32,6 +33,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'edx_theme'
```

Comment:
Given that the edx theme is not actually the right theme for us, let's just not use it from the start.  If we don't like the default, let's figure out what we need from a theme and try to use one off the shelf if we can.

See https://openedx.slack.com/archives/C02M89BSV9A/p1639155646062400 for more context.

Reply:
I've removed the edx-theme and replaced it with the rtd-theme which supports custom colors and custom logo.